x-pack/packages/kbn-elastic-assistant-common/impl/llm/actions_client_llm.ts

```diff
@@ -27,6 +28,7 @@ interface ActionsClientLlmParams {
   model?: string;
   temperature?: number;
   traceId?: string;
+  traceOptions?: TraceOptions;
```

Comment:
Again no change necessary here, but we should look to consolidate `traceId` into `traceOptions` as this is getting a little confusing. The existing `traceId` is intended to allow users of the `ActionsClientLLM` to link any calls together so there's a single `traceId` for all calls initiated from a single request from the kibana client. Again, I've made a note of this for when I work some eval changes here soon. We'll also want to standardize with the new usage of the `ActionsClientChatOpenAI` over `ActionsClientLLM`.